modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -3028,6 +3028,11 @@ <h4><a id="text">Text</a></h4>
           <td>&nbsp;</td>
         </tr>
         <tr>
+        <th class="propertyname" scope="row">-fx-tab-size</th>
+          <td class="value"><a href="#typenumber" class="typelink">&lt;integer&gt;</a></td>
+          <td>8</td>
+          <td>&nbsp;</td>
+        </tr>        <tr>
```

Comment:
the `<tr>` should be on the next line (once you do, you can remove the trailing white space that will then be present).